src/encoding.rs

```diff
@@ -195,29 +195,26 @@ pub struct DecodeContext {
     /// How many times we can recurse in the current decode stack before we hit
     /// the recursion limit.
     ///
-    /// The recursion limit is defined by `RECURSION_LIMIT` and cannot be
-    /// customized. The recursion limit can be ignored by building the Prost
-    /// crate with the `no-recursion-limit` feature.
+    /// It defaults to 100 and can be changed using `prost_build::recursion_limit`,
```

Comment:
So because this is hidden this doc won't actually be readable. So we need to make sure this is documented at the lib level of `prost`.